spec/schemas/_common.yaml

```diff
@@ -686,7 +686,10 @@ components:
             $ref: '#/components/schemas/NodeId'
     NodeId:
       type: string
-    Bytes:
+    StorageSize:
```

Comment:
Related to my comment on `StorageType`, this doesn't seem accurate to me, not all things measured in bytes are storage.

Reply:
What would you prefer? My suggestion would be `BytesHuman` or `BytesString` ?

Reply:
I feel `BytesString` could be mis-interpreted as some mechanism for encoding raw-bytes as a string (ie. a hex/base64 string). Which also kind of applies to `Bytes` on it's own too.

Maybe a more verbose `ByteCount` and `HumanReadableByteCount` would make sense?

Reply:
cc: @dblock @nhtruong ^^

Reply:
The original Elastic docs call these "human-formatted", so I think we could use that term, `HumanFormattedBytes`, or since these appear a lot in the CAT API, which is "compact and aligned text", we could use `BytesText`. WDYT?

Reply:
Just as a side note for CAT API.
Due to the `bytes` parameter the content differ. Without `bytes` parameter it would be printed as `"1gb"` and with it as `"1024"`. So it would not match the `StorageSize`. We would need an extra Type where the unit suffix is optional.

Reply:
The type can have a `anyOf:` with different configurations, one of which is a string with all numbers?